packages/dialog/src/vaadin-dialog.js

```diff
@@ -9,7 +9,7 @@ import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
 import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
 import { processTemplates } from '@vaadin/component-base/src/templates.js';
 import { OverlayElement } from '@vaadin/vaadin-overlay/src/vaadin-overlay.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { ThemePropertyMixin } from '@vaadin/vaadin-themable-mixin/vaadin-theme-property-mixin.js';
```

Comment:
What is the reason behind the dialog importing `ThemePropertyMixin` instead of `ThemableMixin`? Shouldn't it import `ThemableMixin` as others do?

Reply:
Well, I see that the notification component is also using `ThemePropertyMixin` so that it seems not to be a mistake.

Reply:
The reason is that dialog and notification propagate `theme` attribute to overlay, but do not use it themselves.